docs/sql/statements/update.rst

```diff
@@ -53,3 +59,13 @@ Parameters
 :condition:
   An expression that returns a value of type boolean. Only rows for
   which this expression returns true will be updated.
+
+:output_expression:
+
+	An expression to be computed and returned by the UPDATE command after each
```

Comment:
minor: we should be using 4 spaces for indents instead of tabs

(here and in the next indented section)